response_operations_ui/__init__.py

```diff
@@ -32,7 +48,18 @@ def __call__(self, environ, start_response):
 
 
 def create_app(config_name=None):
+    csp_policy = copy.deepcopy(CSP_POLICY)
     app = Flask(__name__)
+    csrf = CSRFProtect(app)
+    csrf.init_app(app)
+    Talisman(
+        app,
+        content_security_policy=csp_policy,
+        content_security_policy_nonce_in=['script-src'],
+        force_https=False,  # this is handled at the firewall
```

Comment:
that comment isn't technically correct, the load balancer is handling it

Reply:
corrected